### ISSUE TITLE
Add support for explaining song choice

### ISSUE THREAD

**Issue by nlowe** [nlowe/mousiki#18]
Pianobar has a feature where you can ask pandora why it picked a track. We should add support to at least do this on-demand. Perhaps we want to add another row to the transport widget to always show it?

**Comment by nlowe** [comment]
The new REST API Might not support this. I'm not seeing an endpoint for it in https://6xq.net/pandora-apidoc/rest/endpoints/. We may have to go rip apart the android app.

**Comment by nlowe** [comment]
Looking at the web UI, this appears to be available at `https://www.pandora.com/api/v1/playlist/narrative`:

```bash
curl -X POST 'https://www.pandora.com/api/v1/playlist/narrative' \
  -H 'accept: application/json, text/plain, */*' \
  -H 'x-authtoken: ***' \
  -H 'x-csrftoken: ***' \
  --data-binary '{"musicId":"S5713101","stationId":"445386914731145560"}' \
  --compressed
```

Response:

```json
{
  "intro":"Based on what you've told us so far, we're playing this track because it features specific musical qualities.",
  "focusTraits":[],
  "paragraph":"Based on what you've told us so far, we're playing this track because it features specific musical qualities."
}
```

However, sometimes it doesn't seem to include any useful information. `https://www.pandora.com/api/v4/catalog/getDetails` seems to have more info but will need some parsing

```bash
curl 'https://www.pandora.com/api/v4/catalog/getDetails' \
  -H 'accept: application/json, text/plain, */*' \
  -H 'x-authtoken: ***' \
  -H 'x-csrftoken: ***' \
  --data-binary '{"pandoraId":"AR:175180"}' \
  --compressed
```

https://gist.github.com/nlowe/f4e221120dcb10965536b3d6a2c0653e

**Comment by nlowe** [comment]
Implemented in e35fe91

![image](https://user-images.githubusercontent.com/794251/91533587-faf7cf80-e8c4-11ea-84e2-c1a7132fa6dc.png)
